README.md

```diff
@@ -1 +1,417 @@
+
+# Introduction
+
+For easier use in reactive user interfaces, especially with `SwiftUI` and `Combine`, I've created a library which abstracts and maps common connectivity APIs. Particularly in my app Heartwitch, I mapped the functionaly of _WatchConnectivity_ and _Network_ over to track the users ability to connect to the Internet as well as the ability for their iPhone to connect to their Apple Watch via _WatchConnectivity_
+
+# Features 
+
+Here's what's currently implemented with this library:
+
+- [x] Monitor network connectivity and quality
+- [x] Communicate between iPhone and Apple Watch
+  - [x] Monitor connectivity between devices
+  - [x] Send messages back and forth between iPhone and Apple Watch
+  - [x] Abstract messages for easier _encoding_ and _decoding_
+
+# Installation
+
+Swift Package Manager is Apple's decentralized dependency manager to integrate libraries to your Swift projects. It is now fully integrated with Xcode 13.
+
+To integrate **SundialKit** into your project using SPM, specify it in your Package.swift file:
+
+```swift    
+let package = Package(
+  ...
+  dependencies: [
+    .package(url: "https://github.com/brightdigit/SundialKit.git", from: "0.2.0")
+  ],
+  targets: [
+      .target(
+          name: "YourTarget",
+          dependencies: ["SundialKit", ...]),
+      ...
+  ]
+)
+```
+
+# Usage 
+
+## Listening to Networking Changes
+
+In the past `Reachability` or `AFNetworking` has been used to judge the network connectivity of a device.**SundialKit** uses the `Network` framework to listen to changes in connectivity providing all the information available.
+
+**SundialKit** provides a `NetworkObserver` which allows you the listen to variety of publishers related to the network. This is especially useful if you are using `SwiftUI` in particular. With `SwiftUI`, you can create an `ObservableObject` which contains an `NetworkObserver`:
+
+```swift
+import SwiftUI
+import SundialKit
+
+class NetworkConnectivityObject : ObservableObject {
+  // our NetworkObserver
+  let connectivityObserver = NetworkObserver()
+  
+  // our published property for pathStatus initially set to `.unknown`
+  @Published var pathStatus : PathStatus = .unknown
+
+  init () {
+    // set the pathStatus changes to our published property
+    connectivityObserver
+      .pathStatusPublisher
+      .receive(on: DispatchQueue.main)
+      .assign(to: &self.$pathStatus)
+  }
+  
+  // need to start listening
+  func start () {
+    self.connectivityObserver.start(queue: .global())
+  }
+}
+```
+
+There are 3 important pieces:
+
+1. The `NetworkObserver` called `connectivityObserver`
+2. On `init`, we use `Combine` to listen to the publisher and store the each new `pathStatus` to our `@Published` property.
+3. A `start` method which needs to be called to start listening to the `NetworkObserver`.
+
+Therefore for our `SwiftUI` `View`, we need to `start` listening `onAppear` and can use the `pathStatus` property in the `View`:
```

Comment:
Does any cleanup need to happen in onDisappear?

Reply:
you could `cancel` I suppose if it's no longer needed but in many cases it may just be run as a singleton. I am to opinions.